src/components/course/services/courses.service.v2.js

```diff
@@ -0,0 +1,20 @@
+const { authenticate } = require('@feathersjs/authentication');
+const coursesUC = require('../uc/courses.uc');
```

Comment:
If we want adding a service if we not really need it and if they include only many TODOs? I think we can remove this file for this pr.